src/components/accessories/patientDataForm/PatientDataForm.tsx

```diff
@@ -37,10 +37,14 @@ const PatientDataForm: FunctionComponent<TProps> = ({
   const { t } = useTranslation();
 
   const validationSchema = object({
-    firstName: string().required("This field is required"),
-    secondName: string().required("This field is required"),
-    birthDate: string().required("This field is required"),
-    sex: string().required("This field is required"),
+    firstName: string().required(t("common.required")),
+    secondName: string().required(t("common.required")),
+    birthDate: string().required(t("common.required")),
+    sex: string().required(t("common.required")),
+    telephone: string().matches(
+      /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/,
+      t("common.incorrectformat")
```

Comment:
It seems fine but as I said this morning, we must test all the different number format.
If it doesn't,we could simple add a Regex that check if the charaters inserted are only numbers.

**Suggestion:**
Maybe we could add some extra information below the fields as for example: info about the accepted phone numbers formats.